pkg/ublox/ublox.go

```diff
@@ -31,7 +31,7 @@ const (
 	// CMD_PROTO_VERSION ...
 	CMD_PROTO_VERSION = " -p MON-VER"
 	// CMD_VOLTAGE_CONTROLER ...
-	CMD_VOLTAGE_CONTROLER = " -v 1 -z CFG-HW-ANT_CFG_VOLTCTRL %d"
+	CMD_VOLTAGE_CONTROLER = " -v 1 -z CFG-HW-ANT_CFG_VOLTCTRL, %d"
```

Comment:
This won't work. There should not be a space after `,`.

Reply:
This can go in the ptpconfig?